agent/lib/src/commands/ci.dart

```diff
@@ -228,6 +233,16 @@ class ContinuousIntegrationCommand extends Command {
     }
   }
 
+  Future<Null> _unlockKeyChain() async {
```

Comment:
Explain why in comments:
1. Xcode build requires the login keychain be unlocked in order to access the certificate for code signing.
2. This seems mitigates the issue of "Your session has expired", see https://github.com/flutter/flutter/issues/17860.

Reply:
Done!